<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputeProfilesConfiguration</class>
 <widget class="QWidget" name="ComputeProfilesConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration de la génération de profils</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0">
     <item row="8" column="1" colspan="3">
      <widget class="QSpinBox" name="spinBoxGenStep">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="value">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Pas entre deux profils</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QCheckBox" name="checkBoxZValuesInPourcent">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Valeurs en pourcentage</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>et &lt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;La configuration par défaut calcule le profil sur toute la grille suivant l'axe Z&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QComboBox" name="comboBoxGen">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Y</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spinBoxMinZ">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Définir l'index de la colonne de la grille à partir de laquelle générer le premier profil&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QSpinBox" name="spinBoxMinX">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Axe principal</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxMinValue">
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>N'utiliser que les cellules dont la valeur est &gt;=</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>et &lt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QSpinBox" name="spinBoxMaxX">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="value">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QComboBox" name="comboBoxOrdonnee">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Z</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Axe de génération</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="4">
      <widget class="QCheckBox" name="checkBoxPersonalized">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Restreindre le calcul sur une portion de la grille</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxValue">
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
       <property name="value">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="3">
      <widget class="QSpinBox" name="spinBoxMaxY">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="value">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QSpinBox" name="spinBoxMaxZ">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Définir l'index de la colonne de la grille de fin de génération&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="value">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Borne de l'axe de génération                    &gt;=</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Bornes de l'axe des ordonnées                 &gt;=</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButtonDefaultConfiguration">
       <property name="text">
        <string>Défaut (RAZ)</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Bornes de l'axe des abscisses                   &gt;=</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="spinBoxMinY">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>et &lt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QRadioButton" name="radioButtonAllGrid">
       <property name="text">
        <string>Sur toute la grille</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QRadioButton" name="radioButtonPersonalized">
       <property name="text">
        <string>Personnalisée</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>et &lt;=</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="checkBoxXYValuesInPourcent">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Valeurs en pourcentage</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>400</width>
        <height>301</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>500</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../resources.qrc">:/resources/computeprofiles.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Axe principal du profil (ordonnée)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>190</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Axe de génération des profils (normal)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Axe abscisses</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Inférieure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>76</x>
        <y>300</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Supérieure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>300</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Inférieure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Supérieure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>266</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pas entre deux profils</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
